cli/collection_create.go

```diff
@@ -17,14 +17,25 @@ import (
 	"github.com/spf13/cobra"
 
 	"github.com/sourcenetwork/defradb/client"
+	"github.com/sourcenetwork/defradb/internal/db"
 )
 
 func MakeCollectionCreateCommand() *cobra.Command {
 	var file string
+	var shouldEncrypt bool
 	var cmd = &cobra.Command{
-		Use:   "create [-i --identity] <document>",
+		Use:   "create [-i --identity] [-e --encrypt] <document>",
```

Comment:
question: should the query cli command also have an encrypted flag?

Reply:
the flag is relevant only when creating a doc. Later if we add it, for example, to update docs, then we can move it somewhere else